how-to-docs/cert-manager.md

```diff
@@ -0,0 +1,235 @@
+## **Cert Manager**
+----
+
+The ACP platform presently has two certificate management services, the first contender was [kube-cert-manager](https://github.com/PalmStoneGames/kube-cert-manager). The project due to a forever changing landscape gradually became deprecated and has now been replaced by the [cert-manager](https://github.com/jetstack/cert-manager). Note ACP will continue to support the kube-cert-manager and the internal cfssl service while they are still in use, we do however recommend shifting over to the cert-manager as aside from security fix we won't be performing anymore updates on these services.
```

Comment:
I'd suggest:
- splitting the second sentence after comma to improve readability. 
- make 'security fix' plural, and add a comma after it.